src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/441e715f-0ccb-41ac-a781-c7e140209730)

Reply:
Changed. Thank you

Reply:
Please, review commit 393df7e, thank you so much